src/bundle/Resources/views/admin/section/view.html.twig

```diff
@@ -25,20 +25,22 @@
             <div class="ez-table-header__headline">{{ 'section.information.header'|trans|desc('Section information') }}</div>
 
             <div>
-                {% if deletable %}
-                    <button type="button" class="btn btn-danger" data-toggle="modal"
-                            data-target="#delete-section-modal" title="{{ 'section.action.delete'|trans|desc('Delete Section') }}">
-                        <svg class="ez-icon">
-                            <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#trash"></use>
-                        </svg>
-                    </button>
-                    {% include 'EzPlatformAdminUiBundle:admin/section:delete_confirmation_modal.html.twig' with {'form': form_section_delete} %}
-                {% else %}
-                    <button type="button" class="btn btn-danger disabled" title="{{ 'section.action.delete'|trans|desc('Delete Section') }}">
-                        <svg class="ez-icon">
-                            <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#trash"></use>
-                        </svg>
-                    </button>
+                {% if can_edit %}
+                    {% if deletable %}
+                        <button type="button" class="btn btn-danger" data-toggle="modal"
+                                data-target="#delete-section-modal" title="{{ 'section.action.delete'|trans|desc('Delete Section') }}">
+                            <svg class="ez-icon">
+                                <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#trash"></use>
+                            </svg>
+                        </button>
+                        {% include 'EzPlatformAdminUiBundle:admin/section:delete_confirmation_modal.html.twig' with {'form': form_section_delete} %}
+                    {% else %}
+                        <button type="button" class="btn btn-danger disabled" title="{{ 'section.action.delete'|trans|desc('Delete Section') }}">
```

Comment:
I think we should not repeat this code, it's the same button no matter if we can delete or not. The disabled state on the button should be added when is not deletable.